angular-src/src/app/components/home/home.component.ts

```diff
@@ -9,86 +9,81 @@ import { take } from 'rxjs/operators';
     styleUrls: [ './home.component.scss' ]
 })
 
-export class HomeComponent implements OnInit{
-    ngOnInit(): void {
-        this._search.getYearRange(1900, 2017).subscribe(response => {
-            if(response.result) {
-                this.allMovieResults = response.result;
-                this.setRevenueData(this.allMovieResults);
-                this.setBudgetData(this.allMovieResults);
-                this.setReleaseDateData(this.allMovieResults);
-                this.setKidFriendlyData(this.allMovieResults);
+export class HomeComponent implements OnInit {
+  this.setRuntimeData(2000,2017);
```

Comment:
You have this function twice. This one and one in the ngOnInit. Delete this one.